readme.md

```diff
@@ -62,16 +63,16 @@ body{
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
+$ git init project-repo
+$ cd project-repo
```

Comment:
Think we should flip these steps ^. We first `cd` into our directory, then we `git init` to ensure we are only tracking the project we care about.